eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -235,11 +235,6 @@ stages:
                         displayName: 'Download Artifact: ${{parameters.ArtifactName}}'
                         artifact: ${{parameters.ArtifactName}}
                       # Pull and build the docker image.
-                      - template: /eng/common/pipelines/templates/steps/docker-pull-image.yml
```

Comment:
Where is the other pull happening?

Reply:
Do we need to clean up any of these variables or are they still used for the other pull?

Reply:
It is inside of update-docsms-metadata.yml

Reply:
https://github.com/Azure/azure-sdk-for-java/blob/main/eng/common/pipelines/templates/steps/update-docsms-metadata.yml#L82